src/app/components/search-results/search-results.component.ts

```diff
@@ -185,6 +185,8 @@ export class SearchResultsComponent implements OnInit, OnDestroy {
     const urlPathname = this.isMobile
       ? "/search-results"
       : "/search/search-results";
```

Comment:
Unrelated to this PR, but I see what you mean about the different routes, interesting.